beagle/src/main/kotlin/br/com/zup/beagle/android/utils/WidgetExtensions.kt

```diff
@@ -65,9 +65,11 @@ fun ServerDrivenComponent.handleEvent(
  * @property eventValue is the value that the eventName name has created,
  * this could be a primitive or a object that will be serialized to JSON
  */
-@Deprecated("It was deprecated in version 1.1.0 and will be removed in a future version." +
-    " Use handleEvent without eventName and eventValue or with ContextData for create a implicit context.",
-    ReplaceWith("handleEvent(rootView, origin, actions)"))
+@Deprecated(
+    "It was deprecated in version 1.1.0 and will be removed in a future version." +
+            " Use handleEvent without eventName and eventValue or with ContextData for create a implicit context.",
```

Comment:
```suggestion
            " Use handleEvent without eventName and eventValue or with ContextData to create an implicit context.",
```

Reply:
I just formatted the code this text change has nothing to do with the purpose of PR so it won't be done, I can even revert the formatting in this code snippet.